System/Primitive.h

```diff
@@ -222,6 +222,76 @@ namespace RTE {
 	};
 #pragma endregion
 
+#pragma region Round Box Primitive
+	/// <summary>
+	/// Class used to schedule drawing of box with round corners primitives created from Lua.
+	/// </summary>
+	class RoundBoxPrimitive : public GraphicalPrimitive {
+
+	public:
+
+		float m_CornerRadius; //!< The radius of the corners of the box.
+
+		/// <summary>
```

Comment:
Same stuff about start and end as in PrimitiveMan. Applies to the fill version below too.